### ISSUE TITLE
Yard 0.9.26 from Gemfile.lock no longer found in source (Gemfile.lock outdated)

### ISSUE THREAD

**Issue by tomharrisonjr** [docmeta/rubydoc.info#172]
```shell
$ git clone git@github.com:docmeta/rubydoc.info.git
$ bundle install
Fetching gem metadata from https://rubygems.org/
Fetching gem metadata from https://rubygems.org/.........
Fetching https://github.com/lsegal/yard.git
Your bundle is locked to yard (0.9.26) from https://github.com/lsegal/yard.git (at main@e21c932), but that version can no longer be found in that source. That
means the author of yard (0.9.26) has removed it. You'll need to update your bundle to a version other than yard (0.9.26) that hasn't been removed in order to
install.
```

Builds fine if I delete the `Gemfile.lock` and `bundle install`

**Comment by chozandrias76** [comment]
> ```shell
> $ git clone git@github.com:docmeta/rubydoc.info.git
> $ bundle install
> Fetching gem metadata from https://rubygems.org/
> Fetching gem metadata from https://rubygems.org/.........
> Fetching https://github.com/lsegal/yard.git
> Your bundle is locked to yard (0.9.26) from https://github.com/lsegal/yard.git (at main@e21c932), but that version can no longer be found in that source. That
> means the author of yard (0.9.26) has removed it. You'll need to update your bundle to a version other than yard (0.9.26) that hasn't been removed in order to
> install.
> ```
> 
> Builds fine if I delete the `Gemfile.lock` and `bundle install`

Confirmed working with this diff:
```diff
diff --git a/Gemfile.lock b/Gemfile.lock
index 75f19e0..e018d49 100644
--- a/Gemfile.lock
+++ b/Gemfile.lock
@@ -1,26 +1,26 @@
 GIT
   remote: https://github.com/lsegal/yard.git
-  revision: e21c9323bba88c841a917d0531281b90f27f2378
+  revision: 0a550939f9b422f1bc4f46481749405284546ef9
   branch: main
   specs:
-    yard (0.9.26)
+    yard (0.9.28)
       webrick (~> 1.7.0)
 
 GEM
   remote: https://rubygems.org/
   specs:
     RedCloth (4.3.2)
-    activesupport (6.1.4.1)
+    activesupport (6.1.6.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 1.6, < 2)
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    asciidoctor (2.0.16)
-    benchmark-ips (2.9.1)
+    asciidoctor (2.0.17)
+    benchmark-ips (2.10.0)
     bluecloth (2.2.0)
-    concurrent-ruby (1.1.9)
-    dead_end (1.1.7)
+    concurrent-ruby (1.1.10)
+    dead_end (4.0.0)
     derailed_benchmarks (2.1.1)
       benchmark-ips (~> 2)
       dead_end
@@ -33,81 +33,85 @@ GEM
       rake (> 10, < 14)
       ruby-statistics (>= 2.1)
       thor (>= 0.19, < 2)
-    diff-lcs (1.4.4)
-    ffi (1.15.4)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
     flamegraph (0.9.5)
     get_process_mem (0.2.7)
       ffi (~> 1.0)
-    github-markup (4.0.0)
+    github-markup (4.0.1)
     heapy (0.2.0)
       thor
-    i18n (1.8.10)
+    i18n (1.12.0)
       concurrent-ruby (~> 1.0)
-    json (2.5.1)
-    kramdown (2.3.1)
+    json (2.6.2)
+    kramdown (2.4.0)
       rexml
     maruku (0.7.3)
     memory_profiler (1.0.0)
     mini_histogram (0.3.1)
-    minitest (5.14.4)
-    mustermann (1.1.1)
+    minitest (5.16.2)
+    mustermann (2.0.2)
       ruby2_keywords (~> 0.0.1)
     net-http-persistent (2.9.4)
     nio4r (2.5.8)
+    psych (4.0.4)
+      stringio
     puma (5.6.4)
       nio4r (~> 2.0)
-    rack (2.2.3.1)
-    rack-mini-profiler (2.3.3)
+    rack (2.2.4)
+    rack-mini-profiler (3.0.0)
       rack (>= 1.2.0)
-    rack-protection (2.2.0)
+    rack-protection (2.2.2)
       rack
-    rack-test (1.1.0)
-      rack (>= 1.0, < 3)
+    rack-test (2.0.2)
+      rack (>= 1.3)
     rake (13.0.6)
     rdiscount (2.2.0.2)
-    rdoc (6.3.2)
+    rdoc (6.4.0)
+      psych (>= 4.0.0)
     redcarpet (3.5.1)
     rexml (3.2.5)
-    rspec (3.10.0)
-      rspec-core (~> 3.10.0)
-      rspec-expectations (~> 3.10.0)
-      rspec-mocks (~> 3.10.0)
-    rspec-core (3.10.1)
-      rspec-support (~> 3.10.0)
-    rspec-expectations (3.10.1)
+    rspec (3.11.0)
+      rspec-core (~> 3.11.0)
+      rspec-expectations (~> 3.11.0)
+      rspec-mocks (~> 3.11.0)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.10.0)
-    rspec-mocks (3.10.2)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.10.0)
-    rspec-support (3.10.2)
-    ruby-statistics (2.1.3)
+      rspec-support (~> 3.11.0)
+    rspec-support (3.11.0)
+    ruby-statistics (3.0.0)
     ruby2_keywords (0.0.5)
-    sequel (5.48.0)
-    sinatra (2.2.0)
-      mustermann (~> 1.0)
+    sequel (5.59.0)
+    sinatra (2.2.2)
+      mustermann (~> 2.0)
       rack (~> 2.2)
-      rack-protection (= 2.2.0)
+      rack-protection (= 2.2.2)
       tilt (~> 2.0)
-    skylight (5.1.1)
+    skylight (5.3.3)
       activesupport (>= 5.2.0)
-    sqlite3 (1.4.2)
-    stackprof (0.2.17)
+    sqlite3 (1.4.4)
+    stackprof (0.2.20)
+    stringio (3.0.2)
     syntax (1.2.2)
-    thor (1.1.0)
-    tilt (2.0.10)
-    tzinfo (2.0.4)
+    thor (1.2.1)
+    tilt (2.0.11)
+    tzinfo (2.0.5)
       concurrent-ruby (~> 1.0)
-    version_sorter (2.2.4)
+    version_sorter (2.3.0)
     webrick (1.7.0)
     yard-kramdown (0.0.1)
     yard-rails (0.3.0)
       yard
     yard-sd (0.0.4)
-    zeitwerk (2.4.2)
+    zeitwerk (2.6.0)
 
 PLATFORMS
-  ruby
+  universal-darwin-21
 
 DEPENDENCIES
   RedCloth
@@ -143,4 +147,4 @@ DEPENDENCIES
   yard-sd
 
 BUNDLED WITH
-   2.1.4
+   2.2.17
```

**Comment by lsegal** [comment]
Fixed by #174
